app/services/workers/create-court-reports.js

```diff
@@ -0,0 +1,51 @@
+const logger = require('../log')
+const Promise = require('bluebird').Promise
+
+const Task = require('../domain/task')
+const taskType = require('../../constants/task-type')
+const taskStatus = require('../../constants/task-status')
+const submittingAgent = require('../../constants/task-submitting-agent')
+const mapCourtReports = require('wmt-probation-rules').mapCourtReports
+const getStgCourtReports = require('../data/get-staging-court-reports')
+const insertWorkloadOwnerAndDependencies = require('../insert-workload-owner-and-dependencies')
+const insertCourtReports = require('../data/insert-app-court-reports')
+const createNewTasks = require('../data/create-tasks')
+
+module.exports.execute = function (task) {
+  var workloadBatchSize = task.additionalData.batchSize
```

Comment:
Not really a workload batch size in this context

Reply:
Yip, changed